Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -168,14 +168,10 @@ - (void)presentViewController:(UIViewController *)alertController
       eventLocationRelativeToWindow = eventLocationInWindow;
     }
     origin = [view convertPoint:eventLocationRelativeToWindow fromView:nil];
-  } else if (view) {
-    CGRect superviewFrame = [superview frame];
-    origin = NSMakePoint(NSMidX(superviewFrame), NSMidY(superviewFrame));
-  } else {
-    origin = [NSEvent mouseLocation];
+    [menu popUpMenuPositioningItem:menu.itemArray.firstObject atLocation:origin inView:superview];
```

Comment:
I think this is still wrong. The line above is getting things in view's coordinate system, but then we're presenting from superview's coordinate system. In the ActionSheetIOS module of RNTester I noticed the "Show Action Sheet with anchor" doesn't actually show it from the anchor and that may be related

Reply:
One caveat - it's possible someone was using the anchorRef as a huge view (say a giant text view) and it's important for them to present the action sheet from a specific location in that view. But maybe that's the edge case that a client should need separate code for?